src/tests/unit/test-util.js

```diff
@@ -0,0 +1,126 @@
+
+import sinon from 'sinon';
+import { expect } from 'chai';
+import _ from 'lodash';
+import * as conn from 'api/v2/db/ldap/connection';
```

Comment:
local imports should be separate from package imports. Add a line here between conn and lodash similar to directory-dao-test.js.